### DISCUSSION TITLE
Bluetooth support cross platform?

### DISCUSSION THREAD

**Discussion by sadrirammal** [dotnet/maui#23229] [Q&A] [answered]
Hey there

I'm currently in the process of developing a production app using .net maui and we need Bluetooth in order to connect to an IoT device. It seems like Maui doesn't support this out of the box. Has anyone done this before and is up for a chat?

**Comment by giuseppenovielli** [comment, marked as answer]
https://github.com/dotnet-bluetooth-le/dotnet-bluetooth-le

**Reply by sadrirammal** [reply]
@giuseppenovielli thanks mate :) have you used it before?

**Reply by acaliaro** [reply]
we use this library in a production app connecting the phone to a EV charger. It works.
You can also take a look to

https://github.com/inthehand/32feet
https://github.com/shinyorg/shiny

**Reply by sadrirammal** [reply]
@giuseppenovielli absolute legend :) its working perfectly
